auth-api/tests/unit/api/test_affiliation.py

```diff
@@ -0,0 +1,248 @@
+
+
+token_header = {
+    'alg': os.getenv('JWT_OIDC_ALGORITHMS'),
+    'typ': 'JWT',
+    'kid': os.getenv('JWT_OIDC_AUDIENCE')
+}
+
+
+def get_claims(role: str = Role.BASIC.value):
+    """Return the claim with the role param."""
+    claim = {
+        'jti': 'a50fafa4-c4d6-4a9b-9e51-1e5e0d102878',
+        'exp': 31531718745,
+        'iat': 1531718745,
+        'aud': 'sbc-auth-web',
+        'typ': 'Bearer',
+        'iss': os.getenv('JWT_OIDC_ISSUER'),
+        'sub': 'f7a4a1d3-73a8-4cbc-a40f-bb1145302064',
+        'firstname': 'Test',
+        'lastname': 'User',
+        'username': 'testuser',
+        'preferred_username': 'testuser',
+        'realm_access':
+        {
+            'roles':
+                [
+                    '{}'.format(role)
+                ]
+        },
+    }
+    return claim
+
+
+def get_claims_invalid(role: str = Role.BASIC.value):
+    """Return the claim with the role param."""
+    claim = {
+        'firstname': 'Test',
+        'lastname': 'User',
+        'username': 'troublemaker',
+        'jti': 'a50fafa4-c4d6-4a9b-9e51-1e5e0d102878',
+        'exp': 31531718745,
+        'iat': 1531718745,
+        'iss': 'foobar',
+        'aud': 'sbc-auth-web',
+        'sub': 'barfoo',
+        'typ': 'Bearer',
+        'realm_access':
+            {
+                'roles':
+                    [
+                        '{}'.format(role)
+                    ]
+            },
+        'preferred_username': 'cp1234567 '
+    }
+    return claim
+
+
+def factory_entity_model():
+    entity = EntityModel(business_identifier='CP1234567')
+    entity.save()
+    return entity
+
+
+def factory_org_model(name, session):
+    """Produce a templated org model."""
+    org_type = OrgTypeModel(code='TEST', desc='Test')
+    session.add(org_type)
+    session.commit()
+
+    org_status = OrgStatusModel(code='TEST', desc='Test')
+    session.add(org_status)
+    session.commit()
+
+    preferred_payment = PaymentTypeModel(code='TEST', desc='Test')
+    session.add(preferred_payment)
+    session.commit()
+
+    org = OrgModel(name=name)
+    org.org_type = org_type
+    org.org_status = org_status
+    org.preferred_payment = preferred_payment
+    org.save()
+
+    return org
+
+
+def factory_affiliation_model(entity_id, org_id):
+    affiliation = AffiliationModel(entity=entity_id, org=org_id)
+    affiliation.save()
+    return affiliation
+
+
+def test_add_affiliation(session, client, jwt):
+    """Assert that the endpoint returns 201."""
+    entity = factory_entity_model()
+
+    entity_info = {
+        'entity': entity.id
+    }
+
+    org = factory_org_model(name='My Test Affiliation', session=session)
+
+    token = jwt.create_jwt(get_claims(), token_header)
+    # print("######################## token {}", token)
+    headers = {'Authorization': f'Bearer {token}', 'content-type': 'application/json'}
+    rv = client.post(f'/api/v1/orgs/' + str(org.id) + '/affiliations', data=json.dumps(entity_info),
```

Comment:
This is nitpicky, but you should use string templating here rather than concatenation.  `/api/v1/orgs/{}/affilications'.format(org.id)`